src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -587,7 +587,7 @@ paths:
           required: false
           schema:
             type: array
-            default: "SOURCES,TABLES,RELATIONSHIPS,PROFILE,DATA_PROJECT"
+            default: "SOURCES,TABLES,RELATIONSHIPS,PROFILE,DATA_PROJECT,DUOS"
```

Comment:
Is there a reason we have to set this default value in yaml and redundantly in code?

Reply:
Thank you for looking at this more closely!

@nmalfroy, I am vaguely remembering a recent instance where we were tied to some suboptimal default value behavior in our controllers.  Is that ringing a bell?

Reply:
Yeah, we shouldn't include access_information by default because it can be really slow (particularly on Azure)

Reply:
Ahh that wasn't quite what I was remembering… it was this conversation, which made me more confident in removing the controller's default parameter configuration:
https://github.com/DataBiosphere/jade-data-repo/pull/1345/files/b6cdb6ffdc84d73f507b767b8c4f7619b213dc6c#r966105813

@nmalfroy I would be interested in chatting more with you about the other changes I made here to remove the redundant default list in code:

https://github.com/DataBiosphere/jade-data-repo/pull/1372/commits/2f524751274f617c237f3ff5b9f0f03158fe89de

I didn't fully remove `SnapshotService.retrieveAvailableSnapshotModel(UUID, AuthenticatedUserRequest)` and force the callers to determine for themselves which fields to include because I wasn't confident about the desired behavior in snapshot export (the only non-test caller).  At any rate, I'm content to have clarified the default behavior within the controller!

Reply:
Flip-flopping back… I reverted this removal.  Our version of `swagger-codegen` 2.4.27 does not support mixing in parameters with refs (I glossed over this in the linked comment above).  So the default is not propagating to the generated API.  We've experimented with `swagger-codegen` upgrades that could fix this, but found that other aspects of code generation broke for us.

For now it felt safest to leave the existing behavior.